src/index.jsx

```diff
@@ -110,6 +110,9 @@ build: ${__JF_BUILD_VERSION__}`);
         Events.on(apiClient, 'requestfail', appRouter.onRequestFail);
     });
 
+    // Listen for route changes
+    appRouter.listen();
```

Comment:
Could the call to `listen()` be moved into the constructor also? Eventually I would like to remove app router entirely, so keeping it more self-contained should help with that.